### ISSUE TITLE
Openhwisk patcher doesn't work when final action is bundled

### ISSUE THREAD

**Issue by tripodsan** [epsagon/epsagon-node#132]
The problem is the:

```
const actions = tryRequire('openwhisk/lib/actions.js');
```

which hides the dependency from webpack. so in the final bundle, it cannot find the source.

replacing it with an explicit `require` works:

```
let actions;
try {
    // eslint-disable-next-line global-require,import/no-unresolved
    actions = require('openwhisk/lib/actions.js');
} catch (e) {
    // ignore
}
```

**Comment by tripodsan** [comment]
I'll provide a PR

**Comment by ranrib** [comment]
Moved to https://github.com/epsagon/epsagon-openwhisk

**Comment by tripodsan** [comment]
this would be fixed with #149
